docs/content/topics/administrators/authentication.md

```diff
@@ -11,4 +11,77 @@ aliases:
 
 # Brigade Authentication
 
-TODO
+Brigade utilizes third-party providers for its user authentication strategy.
+We'll go over the available options and their configuration in this document.
+
+Note that Brigade's default mode of root user access precludes functionality
+related to user management or authorization. Therefore, selection of a viable
+third-party auth strategy (and thus disabling root user mode) is a
```

Comment:
Actually, looking for thoughts from @krancour here.  I had always assumed a binary root user mode OR third-party auth, but there isn't anything in the Brigade chart preventing deployment of both modes enabled.  If that's how we'd like it to continue to be, I can remove the `(and thus disabling root user mode)` as that wouldn't strictly be necessary to use third-party auth.

Reply:
You can do both. It's like Linux. root is a thing, but you shouldn't share it-- which is where the 3rd party auth comes in.